### PR TITLE
COMMERCE-13024 Simple Object Custom Fields in Organization Management

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/COMMERCE-13024

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

File: modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/objects/FieldsWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+import {fetch} from 'frontend-js-web';
+import React, {useCallback, useEffect, useState} from 'react';
+
+import FIELDS_MAPPER from './FieldsWrapperConstants';
+
+const FieldsWrapper = ({
+	mode = 'view',
+	namespace,
+	objectData,
+	objectExternalReferenceCode,
```
Comment (andrea-ale-sbarra) [inline modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/objects/FieldsWrapper.tsx:17]: sort

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
just few comments.

Thank you!

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 2]
```diff
@@ -7207,8 +7207,10 @@ feature.flag.COMMERCE-11922.title=Configurable Purchasing Rules
 feature.flag.COMMERCE-12170.description=Provide REST endpoints for currencies.
 feature.flag.COMMERCE-12170.title=Currencies Headless API
 feature.flag.COMMERCE-12192.description=Users can search, edit, and view extended details for users, organizations, and accounts in the Organization Management widget.
-feature.flag.COMMERCE-12192.title=Organization Management Chart Enhancements
+feature.flag.COMMERCE-12192.title=Organization Management Enhancements
 feature.flag.COMMERCE-12754.title=Refunds
+feature.flag.COMMERCE-13024.description=Users can edit and view any custom fields created through the objects framework for users and accounts, via the Organization Management widget.
```
[the remark this answers is not in the document]
Comment (JamesAGarcia) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:7212]: @andrea-ale-sbarra, I would suggest this:

> feature.flag.COMMERCE-13024.description=Users can view and edit custom object fields for users and accounts via the Organization Management widget.

> feature.flag.COMMERCE-13024.title=Organization Management Widget and Custom Object Fields

... though the title is a tad long haha 😅  

Let me know what you think.

**Review by JamesAGarcia — CHANGES_REQUESTED** [review]

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:commerce-functional

**Comment by smottal** [comment]
ci:test:relevant

**Comment by alessiorendina** [comment]
I'm going to force forward this after @smottal said @baharturk approved

**Comment by alessiorendina** [comment]
ci:forward:force

**Pull request closed without merging**